e2e_samples/parking_sensors/scripts/deploy_azdo_pipelines.sh

```diff
@@ -44,31 +44,23 @@ github_sc_name="${PROJECT}-github"
 github_sc_id=$(az devops service-endpoint list --output json |
     jq -r --arg NAME "$github_sc_name" '.[] | select(.name==$NAME) | .id')
 
-createPipeline () {
-    declare pipeline_name=$1
-    declare pipeline_description=$2
-    full_pipeline_name=$PROJECT-$pipeline_name
-    pipeline_id=$(az pipelines create \
-        --name "$full_pipeline_name" \
-        --description "$pipeline_description" \
-        --repository "$GITHUB_REPO_URL" \
-        --branch "$AZDO_PIPELINES_BRANCH_NAME" \
-        --yaml-path "/e2e_samples/parking_sensors/devops/azure-pipelines-$pipeline_name.yml" \
-        --service-connection "$github_sc_id" \
-        --skip-first-run true \
-        --output json | jq -r '.id')
-    echo "$pipeline_id"
-}
+##Functions created in the common.sh script
 
 # Build Pipelines
+ifexistsoverwrite "ci-qa-python" 
```

Comment:
if you accept my suggestion to combine the 2 functions. remove reference to the other function here and below

Reply:
ok. it is fine